package.json

```diff
@@ -111,7 +111,9 @@
         }
       ],
       "@semantic-release/release-notes-generator",
-      "@semantic-release/npm"
+      "@semantic-release/npm",
+      "@semantic-release/changelog",
```

Comment:
For this you might actually need to install an extra plugin.
Also ideally it should happen before `npm` so that the `npm` package also receives the changelog file.

Reply:
It actually even failed 

```
[3:35:37 PM] [semantic-release] › ℹ  Running semantic-release version 15.13.27
[3:35:37 PM] [semantic-release] › ✖  An error occurred while running semantic-release: Error: Cannot find module '@semantic-release/changelog'
Require stack:
- /home/travis/build/contentful/contentful-import/noop.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at resolveFileName (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/node_modules/resolve-from/index.js:29:39)
    at resolveFrom (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/node_modules/resolve-from/index.js:43:9)
    at module.exports (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/node_modules/resolve-from/index.js:46:47)
    at loadPlugin (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/lib/plugins/utils.js:51:82)
    at /home/travis/build/contentful/contentful-import/node_modules/semantic-release/lib/plugins/index.js:17:37
    at Array.reduce (<anonymous>)
    at module.exports (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/lib/plugins/index.js:14:34)
    at module.exports (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/lib/get-config.js:84:35)
    at async module.exports (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/index.js:167:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/travis/build/contentful/contentful-import/noop.js' ]
}
Error: Cannot find module '@semantic-release/changelog'
Require stack:
- /home/travis/build/contentful/contentful-import/noop.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at resolveFileName (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/node_modules/resolve-from/index.js:29:39)
    at resolveFrom (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/node_modules/resolve-from/index.js:43:9)
    at module.exports (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/node_modules/resolve-from/index.js:46:47)
    at loadPlugin (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/lib/plugins/utils.js:51:82)
    at /home/travis/build/contentful/contentful-import/node_modules/semantic-release/lib/plugins/index.js:17:37
    at Array.reduce (<anonymous>)
    at module.exports (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/lib/plugins/index.js:14:34)
    at module.exports (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/lib/get-config.js:84:35)
    at async module.exports (/home/travis/build/contentful/contentful-import/node_modules/semantic-release/index.js:167:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/travis/build/contentful/contentful-import/noop.js' ]
}npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! contentful-import@0.0.0-determined-by-semantic-release semantic-release: `semantic-release`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the contentful-import@0.0.0-determined-by-semantic-release semantic-release script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2020-04-03T15_35_37_647Z-debug.log
```

Not sure why this still passed.

Reply:
but it wont create a changelog if the repo doesnt have one, right?

Reply:
It might, really not sure.
But if we don't need it, let's skip it?